source/Preferences.h

```diff
@@ -110,11 +120,6 @@ class Preferences {
 	static void ToggleAmmoUsage();
 	static std::string AmmoUsage();
 
-	// Date format preferences.
-	static void ToggleDateFormat();
-	static DateFormat GetDateFormat();
-	static const std::string &DateFormatSetting();
```

Comment:
Why are some of these preference specific functions gone and others aren't? ToggleAmmoUsage I imagine might have something to do with the toggle ammo usage key, but what about the others? Could these not all just be changed to use static variables?

Reply:
Those has specific logic that can't be captured as part of the template class - for example, `ToggleVsync()` skips over modes if they are not supported.